zombienet/xcm-native.toml

```diff
@@ -1,51 +0,0 @@
-[settings]
```

Comment:
Why have you merged `native`and `xcm-native` into one single file? If i don't need to test XCM features, I prefer spawning a local environment without AssetHub Rococo.

If you plan on keeping one single `.toml`, I suggest editing the [README](https://github.com/freeverseio/laos/blob/main/zombienet/README.md)